doc/tools/coqrst/notations/TacticNotationsLexer.py

```diff
@@ -0,0 +1,60 @@
+# Generated from TacticNotations.g by ANTLR 4.7
```

Comment:
Should this be checked into the repo?

Reply:
I've been wondering. It was checked in your repo, and it reduces the number of dependencies. So probably convenient during the migration period, but on the longer term, we'll want to build everything from sources, I guess.